packages/@react-aria/overlays/src/usePopover.ts

```diff
@@ -46,7 +46,11 @@ export interface AriaPopoverProps extends Omit<AriaPositionProps, 'isOpen' | 'on
    *
    * @default false
    */
-  isKeyboardDismissDisabled?: boolean
+  isKeyboardDismissDisabled?: boolean,
+  // TODO: note new prop, essentially a passthrough to useOverlay, allows me to control when useOverlay will
+  // try to close the popover even if focus moves outside of the overlay (aka going from submenu to submenu trigger via mouse hover)
+  // TODO: update description to be popover specific
+  shouldCloseOnInteractOutside?: (element: Element) => boolean
```

Comment:
how do we feel about exposing this?

Reply:
talked about this, we are ok with exposing this, just need to be cognizant that this also affects RAC popover